lib/endpoints.js

```diff
@@ -124,3 +124,30 @@ export const usePrefect = (id, active = true) => {
     prefectError: error,
   }
 }
+
+export const useDataflow = (
+  jobName,
+  startTime,
+  stopTime,
+  severity,
+  active = true
+) => {
+  let options = {}
+
+  if (active) {
+    options = { refreshInterval: 1000 }
+  }
+
+  const route = `/api/dataflow/${jobName}?startTime="${startTime}"&stopTime="${stopTime}"&severity="${severity}"`
+
+  console.log('\n\n ROUTE:', route, '\n\n')
+
+  const { data, error } = useSWR(route, jsonFetcher, options)
+
+  console.log('\n\n DATA:', data, '\n\n')
```

Comment:
@cisaacstern, have you tried the following??


```suggestion
 if (error) return <p>failed to load data</p>
  if (!data)
    return (
      <p>Finished loading data </>
    )
  console.log('\n\n DATA:', data, '\n\n')
```

Reply:
my bad... I meant to put this somewhere else... I don't think the `console.log()` will work here due to the asynchronous nature of `useSWR`.